sources/engine/Xenko.Engine/Rendering/Lights/ITextureProjectionShaderGroupData.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Xenko contributors (https://xenko.com)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Xenko.Core.Collections;
+using Xenko.Core.Mathematics;
+using Xenko.Engine;
+using Xenko.Shaders;
+
+namespace Xenko.Rendering.Lights
+{
+    public interface ITextureProjectionShaderGroupData // TODO: Move to separate file! // UPD 04.08.18 - moved.
```

Comment:
Please remove TODO/UPD comment completely, git has history if we need it.

Reply:
Done.